ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/modules/versions/VersionsServer.kt

```diff
@@ -4,21 +4,36 @@ import com.izivia.ocpi.toolkit.common.OcpiModuleServer
 import com.izivia.ocpi.toolkit.common.httpResponse
 import com.izivia.ocpi.toolkit.modules.versions.services.VersionsService
 import com.izivia.ocpi.toolkit.transport.TransportServer
+import com.izivia.ocpi.toolkit.transport.domain.FixedPathSegment
 import com.izivia.ocpi.toolkit.transport.domain.HttpMethod
+import com.izivia.ocpi.toolkit.transport.domain.VariablePathSegment
 
 class VersionsServer(
-    private val service: VersionsService,
-    basePath: String = "/versions"
-) : OcpiModuleServer(basePath) {
+    private val service: VersionsService
+) : OcpiModuleServer("") {
 
     override suspend fun registerOn(transportServer: TransportServer) {
         transportServer.handle(
             method = HttpMethod.GET,
-            path = basePathSegments
+            path = listOf(FixedPathSegment(service.versionsBasePath))
         ) { req ->
             req.httpResponse {
                 service.getVersions()
             }
         }
+
+        transportServer.handle(
+            method = HttpMethod.GET,
+            path = listOf(
+                FixedPathSegment(service.versionDetailsBasePath),
+                VariablePathSegment("versionNumber")
+            )
+        ) { req ->
+            req.httpResponse {
+                service.getVersionDetails(
+                    versionNumber = req.pathParams["versionNumber"]!!
```

Comment:
Sonarcloud was complaining about that. I marked this issue as won't fix because `handle` implementation has the responsability to make sure that specified `pathParams` exist